backends/p4tools/modules/testgen/lib/test_backend.h

```diff
@@ -48,6 +49,8 @@ class TestBackEnd {
         }
     }
 
+    [[nodiscard]] bool needsToTerminate(int64_t testCount) const { return testCount == maxTests; }
```

Comment:
We can be more defensive here

```suggestion
    [[nodiscard]] bool needsToTerminate(int64_t testCount) const { return testCount >= maxTests; }
```

Reply:
This will cause test generation to terminate if 0 is chosen as input instead of looping until all branches have been explored.

Reply:
Ah, ok then.

Reply:
I am not sure if there is a better solution here but I do like 0 as input for unbounded test case generation.